Model/ModelManager.php

```diff
@@ -92,13 +94,27 @@ public function getParentMetadataForProperty($baseClass, $propertyFullName)
                 $parentAssociationMappings[] = $metadata->associationMappings[$nameElement];
                 $class = $metadata->getAssociationTargetClass($nameElement);
             } elseif (isset($metadata->embeddedClasses[$nameElement])) {
-                $parentAssociationMappings = array();
-                $lastPropertyName = $propertyFullName;
-                $class = $baseClass;
+                $parentEmbeddedMappings[] = array(
+                    'class' => $class,
+                    'field' => $nameElement,
+                );
+                $class = $metadata->embeddedClasses[$nameElement]['class'];
```

Comment:
This line corrupted code, when has multi-level embedding (embedded_first.embedded_second.field).
Because impossible to obtain metadata from embedded class in doctrine.
Is this a bug, or am I mistaken?

Reply:
Not sure, can you create an issue, or if you have a fix for this a PR?